web/src/javascript/app.js

```diff
@@ -3,15 +3,26 @@ import {drawGame, createGameCanvas} from "./game";
 import {gameRefresh} from "./constants";
 
 const gameCanvas = createGameCanvas();
-let currentState;
 
-(new Worker("javascript/gameWebWorker.bundle.js")).onmessage = message => currentState = message.data;
+let currentState;
+const networkWorker = new Worker("javascript/gameWebWorker.bundle.js");
+networkWorker.onmessage = message => {
+  // This is to prevent Chrome's GC from deleting the worker.
+  // It's not happening on FF, I'm pretty sure it's a V8 bug.
+  // Also, it could be caused by a mix of V8 and webpack/babel.
```

Comment:
I personally am not a fan of pointer fingers at other products while it may just be something that is undefined by the standard or similar (it might be a bug! It's just that I don't feel like we necessarily proved it here). Could we reword to only mention that we were experiencing unexpected GC issues on Chrome and that this prevents it from happening? What do you think? Am I being too sensitive here? :)